src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+export const Person = (
+  { person: { name, age, sex, isMarried, partnerName } },
+) => {
+  const marriedWife = isMarried === true && sex === 'f';
+  const marriedHusband = isMarried === true && sex === 'm';
```

Comment:
As a suggestion, you can do the following:
```js
const spouse = sex === 'm'
  ? 'husband'
  : 'wife';

Reply:
Good remark, please fix